### DISCUSSION TITLE
Reference Links Options

### DISCUSSION THREAD

**Discussion by SoniHari** [GitbookIO/community#149] [Feature Requests]
### What's your feature/idea?

Currently, reference links included in content cannot be opened in a new tab. A user navigates to another page by clicking the link from the published page that is viewed by the user. A user would like to navigate back to the content from the content that they viewed prior to navigating to the referenced link. There should be options while creating links that allows to add a title to the link as well as an option to open the referenced link content in a new tab. This feature will help users. 

### What's your desired outcome?

Enable to add a title to the referenced link and allow to open the linked content in a new tab.
![image](https://user-images.githubusercontent.com/23718052/226286288-4cdaa018-f252-4fb2-bebe-4073c9df44aa.png)

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by dfinster** [comment]
👍🏻 +1 for this feature also. We have situations when we need a relative link to open in a new tab, and we cannot without making it an [absolute link](https://docs.gitbook.com/content-creation/editor/inline/link). We really do need these to be relative links that open in new tabs. 

Could we have an "Open in new tab" checkbox for both relative and absolute links?
